content/kubermatic/master/tutorials_howtos/external_clusters/_index.en.md

```diff
@@ -5,15 +5,16 @@ weight = 7
 
 +++
 
-This section describes how to add and manage existing GKE, AKS, and EKS clusters. Once an external cluster is added and
-registered within the KKP platform, you may then upgrade the control plane version or scale the nodes.
-You can also connect any other clusters in the KKP via kubeconfig. Connected clusters can only be viewed, not edited. 
+This section describes how to add and manage existing Kubernetes clusters called as External Cluster in KKP.
```

Comment:
Can we change to "known as" or just "called"?

Reply:
Also, if we use small letter for "clusters" then let's use it for "external clusters" too.